src/ripe/base.py

```diff
@@ -54,6 +54,83 @@ class API(
     transport_rule.TransportRuleAPI,
     availability_rule.AvailabilityRuleAPI,
 ):
+    @classmethod
+    def _query_to_spec(cls, query):

```

Comment:
Looks like this was inspired on the ripe-sdk method but looking at the [ripe-core method ](https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/models/mixins/config.py#L534) method it could be missing some normalization when it handles size, gender, meta...